routes/accounts.js

```diff
@@ -19,12 +19,15 @@ router.get( '/register', ( request, response ) => {
 
 router.post( '/register', ( request, response , next) => {
   const User = request.app.get( 'models' ).User
+  const Item = request.app.get( 'models' ).Item
 
   const { email, password } = request.body
 
   User.create({ email, password: encryptPassword( password ) })
+    .then( user => Item.create({ is_root: true, parent_id: 0, title: 'Home', description: 'Welcome to Floworky', user_id: user.id }))
```

Comment:
This lines exceeds the 80 character limit.  I would also consider refactoring this behavior into a function:
```js
const createRootItem = Item => user => {
  return Item.create({ 
    is_root: true,
    parent_id: 0,
    title: 'Home',
    description: 'Welcome to Floworky'
    user_id: user.id
  })
}
```
Then, this line could be updated as:
```js
  .then( createRootItem( Item ) )
```